#toggles gaze following behavior
bool follow_frame

#the name of the frame to follow
string frameid
